bot/controllers/broadcast.js

```diff
@@ -0,0 +1,69 @@
+module.exports = () => {
+  const Discord = require('discord.js');
+  const messageMiddleware = (message) => {
+    const container = {};
+    container.parsed = message.content.split(' ');
+    const msg = Object.assign(message, container);
+    return msg;
+  };
+  // send message for all channels
+  const _allChannels = (message) => {
+    const msg = messageMiddleware(message);
+    const input = msg.parsed.join(',').replace(/,/g, ' ');
```

Comment:
Semantically, input is a bit too generically named.

Reply:
Would suggest a name like `userMessageInput`.

Reply:
or `usrMsgInput`.

Reply:
Thanks for the suggestion @joegreen2!